integration/src/test/java/org/mule/test/routing/SynchronizedFlowRefOnMuleContextStartTestCase.java

```diff
@@ -88,37 +81,12 @@ private void prePopulateObjectStore() throws ObjectStoreException {
     objectStore.store(testMuleEvent.getCorrelationId(), testMuleEvent);
   }
 
-  public static class UnblockProcessingSource implements MessageSource, Startable {
+  public static class UnblockProcessingSource extends SkeletonSource {
 
     @Override
     public void start() throws MuleException {
       waitMessageInProgress.release();
```

Comment:
don't need this latch, checking for the started flag in skeleton source should be good